### ISSUE TITLE
Two key points of training multispeaker mellotron

### ISSUE THREAD

**Issue by WelkinYang** [NVIDIA/mellotron#112]
When I discovered that two years after Mellotron was published, there were still people who could not successfully train multi-speaker Mellotron, I realized that I had to tell you the two most important points
1、trimming front and end silence of the training data
2、increase the dropout rate of attention and decoder to 0.2 or more

when your setting is right, the alignment will appear with only 20,000 steps

![image](https://user-images.githubusercontent.com/18493279/157580373-766dc8d6-1e4c-4847-8647-6bd1a61cf886.png)
above is multispeaker mandarin mellotron

#22 
#31

**Comment by tenebo** [comment]
Did you get that result by using a Mandarin pre-trained model?
